Lesson19/DynamicIntArray.cpp

```diff
@@ -0,0 +1,87 @@
+#include "DynamicIntArray.h"
+#include <iostream>
+
+// Default c-tor
+DynamicIntArray::DynamicIntArray() :m_size(5), m_position(0) {
+    m_elements = new int[m_size];
+}
```

Comment:
if you're doing in this way, you should rename m_size to m_capacity, m_position to m_size

Reply:
ok, i removed 5. now default value is 0.